java/private/create_jvm_test_suite.bzl

```diff
@@ -72,19 +72,23 @@ def create_jvm_test_suite(
     nontest_srcs = [src for src in srcs if not _is_test(src, test_suffixes)]
 
     if nontest_srcs:
+        lib_dep_label = ":%s-test-lib" % name
```

Comment:
could you create a `lib_dep_name` and use it in the `define_library()` as well. Fewer duplicate of the same code.